util/client/ngmclient/client.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package ngmclient
```

Comment:
Maybe call it `ngmclient.Proxy` is better, as it is actually prodiving proxies instead of client APIs..